_posts/2016-11-15-v2-plan.md

```diff
@@ -27,8 +27,9 @@ This will change the way how to develop Chainer and CuPy, and also how to instal
 
 The version 2 will be developed in a different style from the usual minor updates:
 
-- The development will run on v2 branch, and keep the master branch developing the currently running v1.
-- The v2 branch will use a new CuPy repository.
+- The development will run on `_v2` branch, and keep the master branch developing the currently running v1.
+	- (Update) The branch is renamed to `_v2` to avoid problems around readthedocs.
```

Comment:
How about this? -> The branch `v2` is renamed to `_v2`
Becuase nobody can know the previous name from this statement.